File: project/hyeongwon-onboarding/src/main/java/com/example/hyeongwononboarding/domain/survey/repository/SurveyResponseRepository.java
package com.example.hyeongwononboarding.domain.survey.repository;

import com.example.hyeongwononboarding.domain.survey.entity.SurveyResponse;
import org.springframework.data.jpa.repository.JpaRepository;

/**
 * 설문 응답 Repository
 */
public interface SurveyResponseRepository extends JpaRepository<SurveyResponse, String> {
}

File: project/sungmin-onboarding/survey-service/src/main/kotlin/com/okdori/surveyservice/repository/SurveyResponseRepository.kt
package com.okdori.surveyservice.repository

import com.okdori.surveyservice.domain.SurveyResponse
import org.springframework.data.jpa.repository.JpaRepository

/**
 * packageName    : com.okdori.surveyservice.repository
 * fileName       : SurveyResponseRepository
 * author         : okdori
 * date           : 2025. 6. 18.
 * description    :
 */
interface SurveyResponseRepository: JpaRepository<SurveyResponse, String> {
}
